src/Entity/File.php

```diff
@@ -87,7 +87,7 @@ public function setDimensions(?array $dimensions): void
      */
     public function getWidth(): ?int
     {
-        return null !== $this->dimensions ? $this->dimensions[0] : null;
+        return isset($this->dimensions[0]) ? $this->dimensions[0] : null;
```

Comment:
Please let's simplify this: `return $this->dimensions[0] ?? null`

Reply:
*facepalm* obviously...